src/core/config.cpp

```diff
@@ -226,7 +226,7 @@ void BruceConfig::setRotation(int value) {
 
 
 void BruceConfig::validateRotationValue() {
-    if (rotation!=1 && rotation!=3) rotation = 1;
+    if (rotation<0 && rotation>3) rotation = 1;
```

Comment:
You should've changed the && to an ||

Reply:
Rotation on Portrait are 0 and 2.. I just checked the range from 0 to 3

Reply:
But there's no way rotation to be less than zero AND greater than 3, it has to be OR